linux_os/guide/system/accounts/accounts-session/accounts_polyinstantiated_tmp/oval/shared.xml

```diff
@@ -4,6 +4,7 @@
     <criteria operator="AND" comment="Check Polyinstantiation of /tmp Directories">
       <criterion comment="Check that /tmp-inst exists and has mode 000" test_ref="test_tmp_inst" />
       <criterion comment="Check configuration of /tmp in /etc/security/namespace.conf file" test_ref="test_tmp_in_namespace_conf" />
+      <criterion comment="Check SELinux boolean polyinstantiation_enabled is enabled" test_ref="test_accounts_polyinstantiated_tmp_sebool" />
```

Comment:
Should the check and remediation for sebool `polyinstantiation_enabled` be present in multiple rules?
It makes more sense embed it on `enable_pam_namespace`.
It could also become a rule of its own.

Reply:
Yes, but we already have a rule for this selinux boolean, it's a templated rule, its ID is sebool_polyinstantiation_enabled https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/selinux/selinux-booleans/sebool_polyinstantiation_enabled/rule.yml. But we're adding this rule to the profile.

I like the idea about embedding the enable_pam_namespace, but I would even go firther and merge the check from enable_pam_namespace into accounts_polyinstantiated_tmp and accounts_polyinstantiated_var_tmp.

What is your advise?

Reply:
Oh, I missed that a rule for sebool `polyinstantiaion_enabled` already existed, and it uses an XCCDF Value for the value.

In this case, just selecting it with the appropriate value should do, right?

Reply:
Sorry for the confusion and extra work.

Reply:
It should be enough when the profile selects all 4 rules.